base_layer/core/src/proof_of_work/difficulty.rs

```diff
@@ -56,7 +56,7 @@ impl Difficulty {
 
 impl Default for Difficulty {
     fn default() -> Self {
-        Difficulty(0)
+        Difficulty(1) // Must be more than zero or the geometric mean will fail.
```

Comment:
Makes sense; but there's a comment in [proof_of_work.rs#L114] that is now invalid

Reply:
Comment on ln 114 changed and modified this to use the min difficulty.